### DISCUSSION TITLE
how can i set baseurl for zitadel web ui?

### DISCUSSION THREAD

**Discussion by chezgi** [zitadel/zitadel#6757] [General]
i want to host my zitadel instance in url: http://test.com/zitadel/
i cant find any config in zitadel.
is this possible or i must have an special domain name for my zitadel server?

**Comment by hifabienne** [comment]
hei @chezgi 
Do you use zitadel.cloud or selfhosted? 
In the cloud you can add a custom domain. You can find the documentation here: https://zitadel.com/docs/guides/manage/cloud/instances#add-custom-domain

For self hosted you can find the documentaiton here: https://zitadel.com/docs/self-hosting/manage/custom-domain

**Reply by chezgi** [reply]
thanks @hifabienne,
i am testing zitadel in my desktop.
i khow its domain config, but i want to change baseUrl prefix not domain name.
smaple: mydomain.com/zitadel --> my zitadel instance.
i want to use a gateway to only redirect special prefix to my zitadel backend.

**Reply by hifabienne** [reply]
@eliobischof Can you help here?

**Reply by eliobischof** [reply]
I'm afraid ZITADEL doesn't support configuring all endpoint paths.
You can try to configure your gateway to direct traffic to ZITADEL at these paths: https://zitadel.com/docs/apis/introduction#api-path-prefixes

**Reply by chezgi** [reply]
thanks @eliobischof,
but it is very useful feature to have a setting for baseUrl, 
all of modern web services have this settings.
